### PR TITLE
Allow cc shift rule unique deleted at  = null

### PR DESCRIPTION
**Pull request by iprasetyaa**
- [ ] Penambahan kondisi validasi unique name & time deleted at null

CC : @jabardigitalservice/jds-backend @samudra-ajri @ayocodingit

### CODE REVIEW

**Review by samudra-ajri — APPROVED** [review]

File: app/Http/Requests/CommandCenterShiftRequest.php [hunk 1]
```diff
@@ -29,14 +29,18 @@ public function rules()
         return [
             'name' => [
                 'required',
-                Rule::unique('command_center_shifts')->ignore($this->command_center_shift),
+                Rule::unique('command_center_shifts')->ignore($this->command_center_shift)->where(function ($query) {
```
Comment (ayocodingit) [inline app/Http/Requests/CommandCenterShiftRequest.php:32]:
```suggestion
                Rule::unique('command_center_shifts')->ignore($this->command_center_shift)->whereNull('deleted_at')
```
coba gini kang @iprasetyaa

Comment (iprasetyaa) [reply app/Http/Requests/CommandCenterShiftRequest.php:32]: Saya coba dulu di local kang

Comment (iprasetyaa) [reply app/Http/Requests/CommandCenterShiftRequest.php]: Kang sudah diperbaiki @ayocodingit https://github.com/jabardigitalservice/reservasi-backend/pull/96/commits/0c8a174da1f126eea75a85eeb694db57c7f43adf

Comment (ayocodingit) [reply app/Http/Requests/CommandCenterShiftRequest.php]: sip lebih clean :+1:

**Review by ayocodingit — APPROVED** [review]

**Pull request merged**